bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -59,7 +62,7 @@ spec:
             {{- toYaml .Values.aggregator.containerPorts | nindent 12 }}
             {{- end }}
             {{- if .Values.metrics.enabled }}
-            - name: metrics
+            - name: tcp-metrics
```

Comment:
Not necessary. Istio just requires the port name to follow its standards in the service.

Reply:
As reported [here](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/) in order for Istio to detect the traffic type either you rely on automatic protocol detection or you can be explicit and specify the traffic type manually.
Wherever the traffic type is known it's considered best practice to avoid relying on automatic protocol detection, so this change is correct.